{
  Throwable exception=null;
  try {
    new RustBuckConfig(new FakeBuckConfig()).getRustCompiler();
  }
 catch (  HumanReadableException e) {
    exception=e;
  }
  assumeNoException(exception);
}
